### DISCUSSION TITLE
Din to Usb

### DISCUSSION THREAD

**Discussion by pmjcontroller1** [shanteacontrols/OpenDeck#248] [Q&A]
Hello mate,

We are trying to connect two controllers through the DIN ports.
We have the configuration in two opendeck L plates.

One of these is connected to the PC via USB, and must transmit through that USB its MIDI data and the data it receives from the second board through the DIN MIDI port.
This image shows the configuration of the pcb
![image](https://github.com/shanteacontrols/OpenDeck/assets/118101014/2d691229-5ec6-43eb-abab-ceff3e3778d3)
This is the second controller that connects from DIN OUT to DIN IN on the First board.
This is connected to USB for power.
![image](https://github.com/shanteacontrols/OpenDeck/assets/118101014/2b007b45-fa84-42d5-83d8-e7bb217d53cb)

The problem is that the board that is connected to the PC USB does not send the data from the second one, can you help me?
If you need more information tell me

**Comment by paradajz** [comment]
I cannot reproduce sadly.

**Comment by pmjcontroller1** [comment]
If you need some tests we can help you if you don't have L pcb. But it's very strange because the configuration seems to be correct.

⁣
paco Lop
Dpto comercial
+34 663390200
www pmjcontroller.es​

**Reply by pmjcontroller1** [reply]
Hi paradajz,

I have checked the din signals, and I can't detect any activity.
May be is something wrong in the code?
Where I can see the code related to the din for the opendeck L?

Regards

**Reply by pmjcontroller1** [reply]
Forget it...
I had a bad electrical connection...
😞
